app.go

```diff
@@ -275,8 +279,8 @@ func newApp(args AppArgs) (rapp *App, rerr error) {
 		log:               args.Log,
 	}
 
-	if !args.DontActivate {
-		if err = res.initScreen(); err != nil {
+	if args.Screen == nil {
+		if err := res.initScreen(); err != nil {
```

Comment:
Hi @hinshun - should this be `args.Screen != nil` ?

Reply:
If `args.Screen == nil` it means the user hasn't given their own screen, so `screen` should be managed by gowid. (Initialized on line 240)

Reply:
Thanks @hinshun , you're right, I'm an idiot. I fixed up termshark to compile against your PR and found that keypresses weren't working for several seconds after the process started. But it was a bug in my (termshark's) use of the tcell.Screen!

Reply:
Haha, no worries. Thanks for the review!